docs/transcription.md

```diff
@@ -14,43 +14,106 @@ Depending on the needs of customers, you can define individual prices. You can a
 
-<img src="/transcription/img/transcriptions-automated-transcriptions.png" alt="automated transcript" width="400"/>
+1. Navigate to **Management :material-menu-right: Customer :material-menu-right: [Customer Name] :material-menu-right: Routing :material-menu-right: Ingress Routing  :material-menu-right: Media :material-menu-right: Transcribe**.
+2. From the drop-down you can select from multiple options like **Disabled**, **1% Sampling**, **5% Sampling**, **25% Sampling**, **50% Sampling** and **Enabled (Always On)**.
 
-Click on <img src="/transcription/img/transcriptions-save.png" alt="save" width="120"/>, and the package will be available with the customer.
+!!! Info
+    Out of all the calls that are accessible, **% Sampling** indicates the proportion of calls that will be transcribed.
 
-In Routing :material-menu-right: Media, you can enable or disable this service.
+<img src="/transcription/img/trans1.png" alt="enable" width="900"/>
 
-<img src="/transcription/img/transcription-enable-disable.png" alt="enable" width="900"/>
+3.**Transcription Duration**: You can set a maximum transcription time (in seconds) for your transcription service per call. Please note, the transcription will stop at the specified time, even if mid-sentence.
 
-### Manual Transcriptions
+#### Confirm your Transcription Service Status
 
-Navigate to the **Logging Page** to access the transcribed calls.
+1. Navigate to **Global :material-menu-right: Transcription**. You will be able to see the screen as shown below.
 
-## Search Transcriptions
+<img src="/transcription/img/trans2.png"/>
+
+2.You will see 3 options:
+
+* **Select**: You can select a particular query profile built using the [Transcription Query Profile](https://bani-transcription--connexcs-docs.netlify.app/transcription/#transcription-query-profile).
+* **Search**: Allows you to search things like the **Call ID**, **Phrases** etc. [Click here](https://docs.connexcs.com/transcription/#search-transcriptions) to know more.
+* [Transcription Query Profile](https://bani-transcription--connexcs-docs.netlify.app/transcription/#transcription-query-profile).
+
+3.Description of Transcription fields:
+
+* **Call ID**: Particular Call ID.
+* **Date**: Date of the call.
+* **Customer Name**: Name of the customer whose transcription is displayed.
+* **Text**: Sentences of the transcribed call.
+* **Leg**: Assigns `0` value to caller, `1` to the callie.
+* **Score**: While searching for the relevance of a document, the system rates it and assigns a score; this helps return the documents that best match the search criteria.
+
+#### Transcription Query Profile
 
-Once the system performs the call transcription process, it's placed into a full-text search engine, available only to you and, (in the future) your customer.
+A Transcription Query Profile is a configuration that defines how transcription services should handle if a certain word or phrase is spoken during the call.
```

Comment:
A Transcription Query Profile is a configuration that defines how the transcription service should handle situations when certain words or phrases are spoken during a call.

Reply:
done